### DISCUSSION TITLE
Summary Row?

### DISCUSSION THREAD

**Discussion by Denis-Step** [material-table-core/core#108] [Ideas]
Hello! I am an avid user of Material Table and would like to contribute to this fork seeing as how mbrn is not watching this anymore. One feature I would like to see is a summary row on the top for Number columns. Maintainers, please let me know the best way to get in touch with you.

Thank you.

**Comment by Domino987** [comment]
Hi, thank you for the offer. We introduced a summary row last month with [this](https://github.com/material-table-core/core/pull/137). Can you try it out and see if it fits your need?
